ketcher-autotests/tests/Structure-Creating-&-Editing/Atom/Atom-Properties/Query-specific/query-specific.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { test } from '@playwright/test';
+import {
+  selectRing,
+  resetCurrentTool,
+  takeEditorScreenshot,
+  waitForPageInit,
+  RingButton,
+  clickInTheMiddleOfTheScreen,
+  getCoordinatesOfTheMiddleOfTheScreen,
+} from '@utils';
+import { getAtomByIndex } from '@utils/canvas/atoms';
+
+test.describe('Click Atom on canvas', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  const testIdForQueryProperties = 'query-properties';
+
+  test('Query Specific under Atom Properties - right click - list of properties', async ({
+    page,
+  }) => {
+    /*
+      Description: Checking if list of properties at Query properties under right click is correct.
+    */
```

Comment:
It would be better to assign numbers and correct descriptions to the test cases for easy identification.